### ISSUE TITLE
[Suggestion] Add ip2location.io API into IP Info Multitool

### ISSUE THREAD

**Issue by ip2location-com** [Hue-Jhan/IP-info-multitool#1]
This is a nice project. 

I would like to introduce our free API at https://www.ip2location.io with free 500 queries daily.

If you find it useful, please free to integrate into your project.

**Comment by Hue-Jhan** [comment]
> This is a nice project.
> 
> I would like to introduce our free API at https://www.ip2location.io with free 500 queries daily.
> 
> If you find it useful, please free to integrate into your project.

yeah i think ill do it, its a very cool api and gives more info than the one i used

**Comment by Hue-Jhan** [comment]
> This is a nice project.
> 
> I would like to introduce our free API at https://www.ip2location.io with free 500 queries daily.
> 
> If you find it useful, please free to integrate into your project.

i added 2 more functions using your api, but the api won't recognize tor nodes, vpns, and sometimes even simple proxies. Your website does instead recognizes them, maybe the api is using old databases idk

**Comment by ip2location-com** [comment]
You are right. For the free API, it can detects open proxies only. For Tor nodes, VPNs and other proxy types, it is available in the Plus or Security Plan.
